src/components/Books.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from './Book';
+import AddBook from './AddBook';
+
+const Books = ({ books }) => (
+  <div>
+    <ul className="book-content">
+      {books.map((book) => <Book key={book.id} book={book} />)}
+    </ul>
+    <AddBook />
+  </div>
```

Comment:
- [OPTIONAL] Kindly make sure your Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) -Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) :
- Correct the width and align properties 🤔  you canremove the left margin applied on each book container (margin: 2.313rem 6.25rem 0;) and the `text-align: center;` properly applied to the form and make sure the header, book, and form have the same width and the same align, Kindly correct this 🎨 🖌️ 👨🏻‍🎨 

- Your Design:

![bookstore](https://user-images.githubusercontent.com/79058364/179008149-688f7f26-a431-4fd6-8e29-4b0a4ba59eb9.png)

- Original Design:

![image](https://user-images.githubusercontent.com/79058364/179007912-d92ff84a-db1c-4e94-a7b6-c4386cd05a83.png)